examples/production/scripts/mark-db-baseline.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Mark current state of database as baseline
+# to prevent a restart from clearing and resetting the database.
+# Check repo:./textrepo-app/db for current migration versions
+
+if [[ -z "$1" ]];
```

Comment:
Iets leesbaarder om de 'then' op deze regel te zetten (achter de if) en dan de echos op dezelfde diepte hieronder? Verder hoeven er geen ';' achter de regels normaliter. Anders gaat het op Java-bash lijken :-)

if [[ -z "$1" ]]; then
  echo 'usage: `./mark-db-baseline.sh <version>`'
  echo 'e.g: `./mark-db-baseline.sh 001` marks file starting with `V001__` as baseline'
  exit
fi

Hrm, in 'edit mode' zie je de indents, in 'render mode' worden de indents weggegooid :-S

Reply:
Opgelost in 643d66f